src/Services/Masa.Auth.Service.Admin/Application/Subjects/Commands/UpdateStaffCommandValidator.cs

```diff
@@ -7,13 +7,6 @@ public class UpdateStaffCommandValidator : AbstractValidator<UpdateStaffCommand>
 {
     public UpdateStaffCommandValidator()
     {
-        RuleFor(command => command.Staff.JobNumber).Required().MaximumLength(20);
-        RuleFor(command => command.Staff.DisplayName).Required().MaximumLength(50);
-        RuleFor(command => command.Staff.Name).ChineseLetter().MaximumLength(20);
-        RuleFor(command => command.Staff.PhoneNumber).Required().Phone();
-        RuleFor(command => command.Staff.Email).Email();
-        RuleFor(command => command.Staff.IdCard).IdCard();
-        RuleFor(command => command.Staff.CompanyName).ChineseLetter().MaximumLength(50);
-        RuleFor(command => command.Staff.Position).ChineseLetterNumber().MaximumLength(20);
+        RuleFor(command => command.Staff).SetValidator(new UpdateStaffValidator());
```

Comment:
For example, a nickname is not required on the server side,please refer to the source code to find out the difference,fix it.